.github/workflows/flaky-tests-stalebot-grooming.yml

```diff
@@ -0,0 +1,23 @@
+name: Flaky test auto-grooming
+# Automatically closes flaky test issues when they are over a week old with no activity.
+on:
+    schedule:
+        - cron: '0 0 * * *'
+
+jobs:
+    stale:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Close Stale Issues
+              uses: actions/stale@v9.0.0
+              with:
+                  only-labels: 'metric: flaky e2e test, team: Vortex'
+                  days-before-stale: 5
+                  days-before-close: 7
+                  stale-issue-label: 'metric: stale flaky e2e test report'
+                  stale-issue-message: 'This test may have just been a one-time failure. It will be auto-closed if no further activity occurs within the next 2 days.'
+                  close-issue-message: 'Auto-closed due to inactivity. Please re-open if you believe this issue is still valid.'
+                  close-issue-reason: 'not_planned'
+                  remove-stale-when-updated: true
+                  exempt-all-assignees: true
```

Comment:
If somebody assigns an issue (to themself or to somebody else), then that issue won't be considered for auto-closure.